### DISCUSSION TITLE
Threaded discussions

### DISCUSSION THREAD

**Discussion by etzm** [AKASHAorg/akasha-core#1510] [Ideas]
Is there a way to reply to replies? Want to answer "x" here but I find no way to do so.
To have a meaningful exchange real threaded conversations are quite important.

https://akasha.ethereum.world/social-app/post/01fpg34a02x3qw4szdbh4jd0sk

**Comment by joe-bye** [comment]
i personally guess that it will be improved in the next test. It seems that the "message reply" function should have reserved an interface, but there is no specific implementation. maybe the akasha team can explain this problem。 least in the console, i dont see any errors, which means it may be an empty class
